### ISSUE TITLE
support folders and feed hierarchy

### ISSUE THREAD

**Issue by jancborchardt** [jeena/FeedMonkey#17]
Currently everything is just one stream. Would be cool to be able to focus on one feed or one folder. Also to be able to view only Starred articles for example.

**Comment by jeena** [comment]
Yeah I thought that someone would want that implemented, I just hoped it wouldn't be that soon ;)

**Comment by jancborchardt** [comment]
:)

**Comment by jeena** [comment]
The first approach to tackle this: https://github.com/jeena/FeedMonkey/pull/41
